index.bs

```diff
@@ -549,18 +510,30 @@ spec: mimesniff; urlPrefix: https://mimesniff.spec.whatwg.org/#
 
       <p>
         The <dfn for='AudioConfiguration' dict-member>contentType</dfn> member
-        represents the MIME type of the audio track.
+        represents the [=MIME type=] of the audio track.
       </p>
 
       <p>
-        To check if a {{AudioConfiguration}} <var>configuration</var> is a
+        To check if a {{AudioConfiguration}} |configuration| is a
         <dfn>valid audio configuration</dfn>, the following steps MUST be run:
         <ol>
           <li>
-            If <var>configuration</var>'s {{AudioConfiguration/contentType}} is
-            not a <a>valid audio MIME type</a>, return <code>false</code> and
+            Let |mimeType| be the result of running [=parse a MIME type=]
+            with |configuration|'s {{AudioConfiguration/contentType}}.
+          </li>
+          <li>
+            If |mimeType| is <code>failure</code>, return <code>false</code> and
             abort these steps.
           </li>
+          <li>
+             If the type of |mimeType| per [[RFC9110]] is neither
+             <code>audio</code> nor <code>application</code>, return
+             <code>false</code> and abort these steps.
+          </li>
+          <li>
+            Create an internal slot on |configuration|, \[[mimeType]], and set
```

Comment:
As above... at the same time, this seems very repetitive with the other algorithm... can we merge these two algorithms and switch on the media type?